tableauserverclient/models/target.py

```diff
@@ -0,0 +1,14 @@
+"""Target class meant to abstract mappings to other objects"""
+class Target():
+    def __init__(self, id, target_type):
+        self.id = id
+        self.target_type = target_type
+
+    def __repr__(self):
+        return "<Target#{id}, {target_type}>"
+
+    def get_type(self):
```

Comment:
Ahh Java snuck into your Python code ;)

```
+class Target():
    def __init__(self, id, target_type):
        self.id = id
        self.target_type = target_type

    @property
    def target_type(self):
        return self.target_type
```

Or you can just leave them as attributes without any protection since the object is very simple:
```
class Target(object):
    def __init__(...):
        self.id = id_
        self.target_type = target_type

    def __repr__(self):
        ...
```